app/src/androidTest/java/com/github/fribourgsdp/radio/activities/GameSettingsActivityTest.kt

```diff
@@ -190,7 +194,7 @@ class GameSettingsActivityTest {
                 hasExtra(GAME_PLAYLIST_KEY, Json.encodeToString(testPlaylist)),
                 hasExtra(GAME_NB_ROUNDS_KEY, ctx.getString(R.string.default_game_nb_rounds).toInt()),
                 hasExtra(GAME_HINT_KEY, false),
-                hasExtra(GAME_PRIVACY_KEY, false),
+                hasExtra(GAME_PRIVACY_KEY, true),
```

Comment:
Nice one, this should avoid the test lobbies appearing in public lobbies right?

Reply:
Yessir